### ISSUE TITLE
ConvertTo-PnPClientSidePage can't find existing Wiki Page

### ISSUE THREAD

**Issue by rjcg916** [SharePoint/sp-dev-modernization#7]
A call of this form:
    ConvertTo-PnPClientSidePage -Identity $page.FieldValues["FileLeafRef"] -Overwrite 
generates a page not found error when the $page (WikiPage) is of the form "How To Use This Library.aspx".

**Comment by jansenbe** [comment]
Hi @rjcg916,

Thanks for your feedback. I've just tried below script and that did work fine for me:

```powershell
Connect-PnPOnline -Url https://bertonline.sharepoint.com/sites/test -Credentials bertonline

# Get all the pages in the site pages library
$pages = Get-PnPListItem -List sitepages

# Iterate over the pages
foreach($page in $pages) 
{ 
        # No need to convert modern pages again
        if ($page.FieldValues["ClientSideApplicationId"] -eq "b6917cb1-93a0-4b97-a84d-7cf49975d4ec" ) 
        { 
            Write-Host `Page $page.FieldValues["FileLeafRef"] is modern, no need to modernize it again`
        } 
        else 
        { 
            # Create a modern version of this page
            Write-Host `Modernizing $page.FieldValues["FileLeafRef"]...`
            $modernPage = ConvertTo-PnPClientSidePage -Identity $page.FieldValues["FileLeafRef"] -Overwrite
            Write-Host "Done" -ForegroundColor Green
        }
}
```
I got below output:

```text
Modernizing Home.aspx ...
Done
Modernizing How To Use This Library.aspx ...
Done
```
Alternatively, can you try below command:

```PowerShell
ConvertTo-PnPClientSidePage -Identity "How to use this library.aspx" -Overwrite
```

**Comment by rjcg916** [comment]
I believe this error is the result of multiple SitePages / Wiki Libraries within a web.  (See #8)

**Comment by jansenbe** [comment]
Indeed. Closing as duplicate of #8. The corresponding bug to fix this is #12
